doc/modules/outlier_detection.rst

```diff
@@ -112,14 +107,33 @@ an ellipse. For more details on the different estimators refer to the example
 :ref:`sphx_glr_auto_examples_plot_anomaly_comparison.py` and the sections
 hereunder.
 
+.. figure:: ../auto_examples/images/sphx_glr_plot_anomaly_comparison_001.png
+   :target: ../auto_examples/plot_anomaly_comparison.html
+   :align: center
+   :scale: 50
+
+The following figure benchmarks outlier detection algorithms using ROC
+(Receiver Operating Characteristics) curve on fives different datasets. 
+:class:`ensemble.IsolationForest` outperforms
+:class:`neighbors.LocalOutlierFactor` because LOF is not meant to predict on
+a test set by default.
+
+.. figure:: sphx_glr_plot_anomaly_bench_001.png
+   :target: ../auto_examples/plot_anomaly_bench.html
+   :align: center
+   :scale: 50
+
 .. topic:: Examples:
 
   * See :ref:`sphx_glr_auto_examples_plot_anomaly_comparison.py`
     for a comparison of the :class:`svm.OneClassSVM`, the
     :class:`ensemble.IsolationForest`, the
     :class:`neighbors.LocalOutlierFactor` and
     :class:`covariance.EllipticEnvelope`.
-
+    
+  * See :ref:`sphx_glr_auto_examples_plot_anomaly_bench.py` for an anomaly
```

Comment:
See ... for an example showing how to evaluate outlier detection estimators using ROC curves.